docs_src/microservices/application/sdk/details/StoreAndForward.md

```diff
@@ -0,0 +1,128 @@
+---
+title: App SDK - Store and Forward
+---
+
+# App Functions SDK - Store and Forward
+
+The Store and Forward capability allows for export functions to persist data on failure and for the export of the data to be retried at a later time. 
+
+!!! note
+    The order the data exported via this retry mechanism is not guaranteed to be the same order in which the data was initial received from Core Data
+
+## Configuration
+
+`Writable.StoreAndForward` allows enabling, setting the interval between retries and the max number of retries. If running with Configuration Provider, these setting can be changed on the fly via Consul without having to restart the service.
+
+!!! example "Example - Store and Forward configuration"
+    ```yaml
+    Writable:
+      StoreAndForward:
+        Enabled: false
+        RetryInterval: "5m"
+        MaxRetryCount: 10
+    ```
+
+!!! note
+    RetryInterval should be at least 1 second (e.g. '1s') or greater. If a value less than 1 second is specified, 1 second will be used. Endless retries will occur when MaxRetryCount is set to 0. If MaxRetryCount is set to less than 0, a default of 1 retry will be used.
+
+Database configuration section describes which database type to use and the information required to connect to the database. This section is required if Store and Forward is enabled. It is optional if **not** using `Redis` for the EdgeX MessageBus which is now the default. 
```

Comment:
the last sentence here - are you trying to say that Store and Forward is optional when not using Redis (which is the default EdgeX Message Bus)? It's a little awkward in the wording - caused me to stumble a bit and think hard about it.

Reply:
Not sure what that last bit was trying to say. Removed it. Also add that this is provided in the common configuration.